docs/version-1.9/modules/en/pages/installation-setup/installation/airgapped-environment.adoc

```diff
@@ -2,363 +2,168 @@
 :current-version: {page-component-version}
 :doctype: book
 
-{longhorn-product-name} can be installed in an air gapped environment by using a manifest file, a Helm chart, or the Rancher UI.
+{longhorn-product-name} can be installed in an air-gapped environment by using a Helm chart.
 
-== Requirements
+== Prerequisites
 
+* https://helm.sh/docs/[Helm] v3.0 or later is required.
 * Deploy {longhorn-product-name} components images to your own registry.
 * Deploy Kubernetes CSI driver components images to your own registry.
 
+== Image Management for Air-Gapped Environments
+
+. **Obtain the list of required images** +
+Run the link:{attachmentsdir}/download-longhorn-image-list.sh[script] to download the complete list of all images required for {longhorn-product-name}. The script will generate a `longhorn-images.txt` file containing the list of images.
+
 [NOTE]
-.Images required and getting them
+====
+Authentication to AppCo is necessary when running this script, as it uses Helm internally. For more information, refer to the https://docs.apps.rancher.io/get-started/authentication/#helm[Helm authentication documentation].
```

Comment:
```suggestion
This script pulls the SUSE Storage chart from the SUSE Application Collection OCI registry to extract the required image list. Helm must be configured with authentication credentials to access the registry. For setup instructions, refer to the [Helm authentication documentation](https://docs.apps.rancher.io/get-started/authentication/#helm).
```

Reply:
Briefly introduce the reason for the required authentication, then add the authentication doc.